### DISCUSSION TITLE
Windows 11, office 365, wordmat.dtom får word til at crashe

### DISCUSSION THREAD

**Discussion by Merbt** [Eduap-com/WordMat#134] [Support]
Hej

Når jeg har installeret Wordmat (nyeste download) og starter Word crasher word og tilføjelsesprogrammet skal deaktiveres inden jeg kan starte word igen.
Jeg har angivet sikkerhedssetuppet i "Center for sikerhed og rettighedsadministration" i windows så c:\programmer (X86)\wordmat og c:\program files\microsoft office\Root\Templates er placeringer der er tillid til. Derudover har jeg sat "Aktiver alle makroer" samt har tillid til VBA-projektmodellen.

Hvis jeg forsøger at åbne wordmat.dotm filen crasher word uden fejl meddelelser.
Jeg har prøvet at deaktivere anti virus og har verificeret at installer filen ikke er blokkeret.

Jeg er lidt lost med hvad der ellers kan gøre at wordmat ikke virker.

Mvh

Jimmi Merbt

**Comment by christinamunch** [comment]
Hej,
Fandt du en løsning på dit problem? Jeg oplever præcis det samme på mit computer lige nu, og jeg kan ikke finde en løsning.

Mvh Christina

**Comment by MortenGraae** [comment]
Jeg har oplevet det på computere hvor malwarebytes er installeret

**Reply by christinamunch** [reply]
Jeg har lige undersøgt om der er malwarebytes på min computer, det er der ikke. Er der andre muligheder?
